kernel_tracing/metric_evaluation/simple_bpftrace.bt

```diff
@@ -0,0 +1,7 @@
+// This is used to demo the metric evaluation script without requiring any
+// dependencies. Run `./test.py no_dependencies.json` to try it, and change this
+// value to see the failure output.
+
+BEGIN {
+    @val = 42;
+}
```

Comment:
I don't get what this is for and what '42' might possibly represent - other that "the answer is always 42"

Reply:
The answer is always 42 for this script. The purpose is to be able to run the python framework without having any of the kernel dependencies for bpf installed.

Reply:
Ah ok, so now I get it.

maybe rename to '@test_check' or something. 'val' threw me off, I didn't understand what this was.